bridge/qb/client/main.lua

```diff
@@ -6,7 +6,11 @@ require 'bridge.qb.client.events'
 local qbCoreCompat = {}
 qbCoreCompat.PlayerData = QBX.PlayerData
 qbCoreCompat.Config = lib.table.merge(require 'config.client', require 'config.shared')
+
 qbCoreCompat.Shared = require 'bridge.qb.shared.main'
+qbCoreCompat.Shared.Jobs = {}
+qbCoreCompat.Shared.Gangs = {}
```

Comment:
Can we not keep these definitions in the main shared file? I'm pretty sure the server needs them too

Reply:
Forgot to add the server side. Fixed.